submissions/kazeemErinfolami/htmlForms/contact.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Contact</title>
+  </head>
+  <body>
+    <form>
+      <div>
+        <h1>Contact Form</h1>
+      </div>
+      <hr />
+      <label for="name"><b>Full Name </b>:</label>
+      <input type="text" placeholder="name" id="name" name="name" /><br /><br />
+      <label for="email"><b>Email</b> :</label>
+      <input type="email" placeholder="email" id="email" name="email" /><br /><br />
+      <label for="subject"><b>Subject </b>:</label>
+      <input type="text" placeholder="Subject" id="subject" name="subject" /><br /><br />
+      <label for="message-input"><b>Text </b>:</label>
+      <textarea placeholder="message input" id="message-input" name="message-input" rows="4" cols="50"
+      ></textarea
```

Comment:
Adjust the closing tags here to be attached to the right attribute @kazeemerinfolami